frontend/src/components/editor/Cell.tsx

```diff
@@ -499,7 +596,13 @@ const CellComponent = (
                 onRun={handleRun}
               />
             </div>
-            <div className="absolute flex flex-col gap-[2px] justify-center h-full left-[-34px] z-20">
+            <div
+              className={cn(
+                "absolute flex flex-col gap-[2px] justify-center h-full left-[-34px] z-20",
+                isMarkdownCodeHidden && "-top-7",
+                isMarkdownCodeHidden && canCollapse && "gap-3", // Add extra gap for collapse button
```

Comment:
Can we slightly increase the gap to create more space between the collapse button and the add cell buttons?

```suggestion
                isMarkdownCodeHidden && canCollapse && "gap-4", // Add extra gap for collapse button
```

Reply:
The borderless view looks really clean too, and I don't think its too difficult of a change.

Reply:
@Light2Dark these both look excellent — I'm happy for them to both land in this PR, will leave it up to @mscolnick. Thanks you!!